sql/core/src/main/scala/org/apache/spark/sql/execution/python/TransformWithStateInPandasExec.scala

```diff
@@ -106,6 +106,30 @@ case class TransformWithStateInPandasExec(
     List.empty
   }
 
+    // operator specific metrics
```

Comment:
Same, shall we move the change for metrics (and test) out to separate JIRA ticket and corresponding PR?